resources/language/resource.language.hr_hr/strings.po

```diff
@@ -518,7 +518,10 @@ msgid "Sync My List to Kodi library"
 msgstr "Sinkroniziraj Moju Listu sa Kodi zbirkom"
 
 msgctxt "#30123"
-msgid "Do you really want to proceed?\nThis will delete all previously exported items and export all items from My List for the currently active profile.\nThis operation may take some time..."
+msgid ""
```

Comment:
do not change structure of msgid

Reply:
Will fix that PoEdit did this automatically when saving the file...

Reply:
This should be fine now